hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/state/WritableKVStateBaseTest.java

```diff
@@ -261,9 +263,93 @@ void getForModifyAfterRemove() {
 
             // Commit should cause the value to be removed
             state.commit();
-            Mockito.verify(state, Mockito.never()).putIntoDataSource(Mockito.anyString(), Mockito.anyString());
-            Mockito.verify(state, Mockito.times(1)).removeFromDataSource(Mockito.anyString());
+            Mockito.verify(state, Mockito.never()).putIntoDataSource(anyString(), anyString());
+            Mockito.verify(state, Mockito.times(1)).removeFromDataSource(anyString());
+            Mockito.verify(state, Mockito.times(1)).removeFromDataSource(A_KEY);
+        }
+    }
+
+    /**
+     * Gives size of backing store plus modifications (additions or removals).
+     * If a new key is added by calling {@code put()}, then size increases, as new key is added to modifications map for addition.
+     * If an existing key is removed by calling {@code remove()}, then size decreases, as new key is added to modifications map for removal.
+     */
+    @Nested
+    @DisplayName("size")
+    final class SizeTest {
+        @Test
+        @DisplayName("Adding a key that does not already exist in the backing store impacts size")
+        void putNew() {
+            assertThat(state.readKeys()).isEmpty();
+            assertThat(state.modifiedKeys()).isEmpty();
+
+            // Before doing put, the size should be 2
```

Comment:
Same thing, magic number? There are a few of these

Reply:
will add comment that 2 will be from set up